### PR TITLE
Brian/bulk shipping

### PR DESCRIPTION
**Pull request by brian31988**
https://stockx-services.atlassian.net/browse/BS-33

Pairs with https://github.com/stockx/freighter/pull/242

### CODE REVIEW

File: lib/active_shipping/carriers/ups.rb [hunk 1]
```diff
@@ -631,6 +631,14 @@ def build_international_forms(xml, origin, destination, packages, options)
       end
     end
 
+    def products(packages)
+      if packages && packages.first && packages.first.options && packages.first.options[:products]
+        packages.first.options[:products]
```
Comment (emmercm) [inline lib/active_shipping/carriers/ups.rb:636]: Is `.first` good enough or should we combine all products in all packages?

Comment (grayxr) [reply lib/active_shipping/carriers/ups.rb:636]: @brian31988 ^^^ Was this comment resolved IRL? Is `packages` always an array with one element?

Comment (brian31988) [reply lib/active_shipping/carriers/ups.rb:636]: Yea packages is always an array with one element

**Review by grayxr — APPROVED** [review]
lgtm

**Pull request merged**